lib/registries/addon/components/editable-field/template.hbs

```diff
@@ -0,0 +1,20 @@
+{{#if @registration.userHasAdminPermission}}
+    <div local-class='Editable'>
+        <h4>{{t @title}}</h4>
+        {{#if @registration.userHasAdminPermission}}
+            <OsfButton
+               data-test-add-institution-button
+               aria-label={{t 'general.add'}}
+               data-analytics-name={{@analyticsName}}
```

Comment:
Instead of requiring passing in `@analyticsName`, put `data-analytics-name='Edit'` (or something) and let the invoker use `data-analytics-scope` to capture which field it is.